x-pack/plugins/index_management/common/constants/plugin.ts

```diff
@@ -17,3 +17,9 @@ export const PLUGIN = {
       defaultMessage: 'Index Management',
     }),
 };
+
+// Ideally we want to access the Kibana major version from core
+// "PluginInitializerContext.env.packageInfo.version". In some cases it is not possible
+// to dynamically inject that version without a huge refactor on the code base.
+// We will then keep this single constant to declare on which major branch we are.
+export const MAJOR_VERSION = '8.0.0';
```

Comment:
This will be changed to `7.16.0` when backported and I will import that same constant for the work done here (https://github.com/elastic/kibana/pull/112990)

Reply:
Should we create a ticket so we don't forget to do so?

Reply:
I will immediately do it after merging 😊

Reply:
Done on the PR itself: https://github.com/elastic/kibana/pull/113512/commits/d45518a9b67a2a53be6ce73dbdcb421cbe4006e9